cluster/examples/kubernetes/ozone/object-store.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ozone.rook.io/v1alpha1
+kind: ObjectStore
+metadata:
+  name: my-store
+spec:
+  ozoneVersion:
+      image: elek/ozone:20190926-1
```

Comment:
is there an official ozone image to point to? I assume this is your private build.

Reply:
Yes, this is a temporary solution, will be changed by the next week.

0.4.1 is released on last weekend. The new image will be published with https://issues.apache.org/jira/browse/HDDS-2025

Reply:
Is the official image published now that we can use here?